test/baseline/User.ts/Products.test.js

```diff
@@ -20,7 +20,9 @@ describe('User.ts/products endpoint test', () => {
     expect(response.status).to.eql(401)
   })
 
-  it('should get user products', async () => {
+  // FIXME: 20221010 Skip. This test currently throws a StripeAuthenticationError
```

Comment:
Hmm, yeah I don't think we want to actually access Stripe in our test suite. We should figure out a way around that.

Reply:
Agreed. We can figure out some sort of mock for this instead of hitting Stripe.